docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -353,14 +353,40 @@ This value is `true` by default.
 
 ### ttl
 
-_Time to live_ (TTL) is the time that a query result is stored in the caching system before it is deleted or refreshed. This setting defines the time to live for query caching, when TTL is not configured in data source settings. The default value is `5m` (5 minutes).
+_Time to live_ (TTL) is the time that a query result is stored in the caching system before it is deleted or refreshed. This setting defines the time to live for query caching, when TTL is not configured in data source settings. The default value is `1m` (1 minute).
```

Comment:
@kminehart just to confirm this change, is this default set to `1m` here? https://github.com/grafana/grafana-enterprise/blob/ea5b5428f9dd8ca336dc53ce9eeb57b17d1eb587/src/pkg/extensions/caching/config/storage_settings.go#L26

Reply:
yes that's where the default is set. I think it was originally set to 5m but I figured that was too much.